examples/platform/silabs/MatterConfig.cpp

```diff
@@ -45,15 +41,6 @@
 #include "MemMonitoring.h"
 #endif
 
-// TODO: We shouldn't need any platform specific includes in this file
-#if (defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1)
-#include <platform/silabs/SiWx917/SiWxPlatformInterface.h>
-#endif // (defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1 )
```

Comment:
This can't be removed for the time being. Can you re-add it with the same platform todo.